projects/trustswap/index.js

```diff
@@ -1,56 +1,80 @@
 const sdk = require("@defillama/sdk");
 const erc20 = require("../helper/abis/erc20.json");
 const abi = require("./abi.json");
+const { transformBscAddress, transformPolygonAddress, transformAvaxAddress } = require("../helper/portedTokens");
 
-const staking_contract = "0x5A753021CE28CBC5A7c51f732ba83873D673d8cC";
+const tokens = {
+  'polygon': '0x3809dcdd5dde24b37abe64a5a339784c3323c44f',
+  'avax': '0xc7b5d72c836e718cda8888eaf03707faef675079',
+  'bsc': '0x82443a77684a7da92fdcb639c8d2bd068a596245',
+  'ethereum': '0xcc4304a31d09258b0029ea7fe63d032f52e44efe'
+};
+const stakingContracts = {
+  'polygon': '0xb16a2a990a4EAdB677505fdF5bdb299f703fce25',
+  'avax': '0x405eF38d44ACfF35b293410fEF9d8de1369Bece4',
+  'bsc': '0x1714FBCFb62A4974C83EaFA0fAEC12191da6c71e',
+  'ethereum': '0x5A753021CE28CBC5A7c51f732ba83873D673d8cC'
+};
 
 const assets = [
   // other tokens which probably for some reason was sent to the contract accidentally

```

Comment:
I dont think we should be counting this, because these arent funds users have chosen to deposit, and they cant withdraw them. Its not really defi